.travis.yml

```diff
@@ -1,41 +1,145 @@
-  # ID
-  - secure: oPl7aI9rCab0Q1qWu+ikMWKsFUhzkvqyXNwrPzeKtg9AifBMGmil2H5A877yajwbU6cyPxmRr8HGrUo4kMoOn8eqaVhSjt41Qi0X11ftx4gjz+MRpSMXso3dKxXsS2vh72B23tm9MqnP8OBkxESEDCBiV7IRIuS9B9zvCEr8jkfLw4BpZy7unyEMkXW3yrPuoehZ4voN7igUgn0+1EqK7aAeev3zCRcehW8rq3Zis+0IOw0KZeF8+kvi56FLXUzR96FfxxS+DoCRiU3T/+o7bKROGabZnc4aCrmMwGiUfGiffNoJSc3OzhqzNubYalRhZyrf//tRWewnQlUtIQzn8CHONwOxQFIw8RzIOwAej05Qr32lOLdtivUxMerokRA3Hc4l0IKeyaR7tMuzHs2T/i/IborBF5rUKBKbVTABvzVhuID79KXTxvp9y1MHdT5tybJLI4CqBH5XuT3GW5LweaDJ5MIIeRRKyQDjq08l1z5Btd65y/FyaZcFGzvkjzgxRj8g1+mXdUwiW6f/FmtFP8T+d9LNxW/30tx9rtPi/XBoi5RyewN88VUAeZq36sgmunLWozLQppdzhtSpDUc2GO48HxYsasuQUOdgQWy24sGFpWtT1/h79Wl1dxUkHTAw5a6OcXYfH6HoWanhSQ20vxBS1ZbcpoVHoqf1RdGJ/XU=
-  # DOCKER_PASSWORD
-  - secure: "IWWZ673TdyxLsvURJUMFt9fzDp7pKMRrZjCHkBqLrJE7fqECfN0WN+rkY2IaVsjGvc5VIa/05/9xYVEIX4rBhtLtZE2YTkXjU/eMjxs+lSHpVkyyLebYJVmCV+f6P4fjNIKNtr+ucMBklExKYaW9LI2noeYXKzyHmWfDmAy7KubFo8J5OHaODYqOQgNKdeX15JAJFH3K5CjhQxY5QBTvuXhZFDwcPeDo526mYiHJBXszpOCCjmEXN/YYT+gEO/jxakCUT4vLaZt9feSY1Sd6Wxm9eGLvYa6B/lwvMt0FWehZ+TSDGXagDyyK8oFqeZojbZyBTjhWj/12qaf3uPiHUU60eMbwaOfqwOZUxuVU+atx/ntxMQO22mr66DyGTcyVET9Yf5nFuVfyTiwPbBIDY4jLMRL/ZcOQDLrX2ObWRqSIvxHh24qzTFqvxR2FBqf+WHvYf4FVDUxmC3GxS5RRZ8n5osHic7tnEKij0lU9TlYfBYn7ESXyWB7iZsj5lw73s9ityXpJo3AfffCWBttAhI/FfDckwyF+vhh0+ULU8mYpgbCCZ3DKob35bZmS97LAE4QBKe22INfKH8Lxzq3qDp5tcbwDCjJUsEF9HP3oBy/F36OO1CZarrVJXDuWi1hf51KtWuEusOdcF4twZn3RVB+7M72prkx+YRh2P5UvQoM="
-  - BUCKET_NAME=dl.kuzzle.io
-  - DOCKER_COMPOSE_VERSION=1.12.0
-  - TAR_EXCLUDES="--exclude=docker --exclude=features --exclude=run_tests.sh --exclude=*node_modules"
+# ------------------------
+# Jobs configuration
+# ------------------------
+jobs:
+  include:
+    # ----------------------------
+    # Deadlinks Tests
+    # ----------------------------
+    - name: Dead link check
+      stage: Tests
+      language: node_js
+      node_js: "10"
+      cache:
+        directories:
+          - "$HOME/.gem/specs"
+          - ".doc/framework/node_modules"
+      before_script:
+        - npm run doc-prepare
+        - npm run --prefix doc/framework clone-repos
+      script:
+        - gem install typhoeus
+        - HYDRA_MAX_CONCURRENCY=20 npm run --prefix doc/framework dead-links
 
-before_install:
-  - sudo sysctl -w vm.max_map_count=262144
-  # Build images
-  - docker login -u aschen -p $DOCKER_PASSWORD
-  - for dir in $(\ls -1d */docker); do sh -c "cd $dir && bash build_images.sh && bash push_images.sh"; done
+    # ----------------------------
+    # Archive deployment
+    # ----------------------------
+    - <<: *deploy-base
+      name: Deploy Archives to dl.kuzzle.io
+      if: branch = master
+      stage: Deployments
 
-addons:
-  apt:
-    packages:
-      - ruby
-      - awscli
+      install:
+        - pip install awscli --upgrade --user
+      before_install:
+        - sudo sysctl -w vm.max_map_count=262144
+        # Build images
+        - docker login -u kuzzleteam -p $DOCKER_PASSWORD
```

Comment:
Missing `DOCKER_PASSWORD` variable in secure env variables

Reply:
I've add it into Travis

Reply:
Hmm ok, usually we prefer to put everything in the `.travis.yml` file so there is no hidden var but I won't block this PR for this